<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label Lobster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <layout class="QVBoxLayout" name="leftColumn">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Drucken und Labelformat</string>
        </property>
        <layout class="QVBoxLayout" name="layoutGroup1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Druckerauswahl</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPrinterSlection">
           <property name="contextMenuPolicy">
            <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="paper_format">
           <property name="text">
            <string>Drucker Papierformat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPrinterPaperSelection"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Versandlabeltyp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbShippingLabelType"/>
         </item>
         <item>
          <widget class="QLabel" name="lblSizeLabel">
           <property name="text">
            <string>Labelgröße</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblLabelSize">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRatioLabel">
           <property name="text">
            <string>Seitenverhältnis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRatio">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Automatik</string>
        </property>
        <layout class="QVBoxLayout" name="layoutGroup3">
         <item>
          <widget class="QPushButton" name="btnSaveCrop">
           <property name="text">
            <string>Als Standard Speichern</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Drucken</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnPrint">
           <property name="text">
            <string>Drucken</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightColumn">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Originales Versandlabel</string>
         </attribute>
         <layout class="QVBoxLayout" name="tab1Layout">
          <item>
           <widget class="QGraphicsView" name="graphicsViewImportedLabel"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionImportShippingSlip"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
    <addaction name="actionShowSupplierLabelManager"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLabelmanager">
   <property name="text">
    <string>Labelmanager</string>
   </property>
  </action>
  <action name="actionShowPaperManager">
   <property name="text">
    <string>Papierverwaltung anzeigen</string>
   </property>
  </action>
  <action name="actionShowSupplierLabelManager">
   <property name="text">
    <string>Versandlabels konfigurieren</string>
   </property>
  </action>
  <action name="actionImportShippingSlip">
   <property name="text">
    <string>Versandlabel importieren</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
